randovania/games/dread/json_data/Cataris.txt

```diff
@@ -701,10 +701,16 @@ Extra - asset_id: collision_camera_012
       Any of the following:
           Varia Suit or After Cataris - Move Magnet Walls Button
           Heat/Cold Runs (Intermediate) and Heat Damage ≥ 100
-  > Event - Activate Magnet Walls Button
-      Shoot Diffusion or Wave
   > Below Thermal Door
       After Cataris - Hallway Thermal Device
+  > Event - Activate Magnet Walls Button (Below)
+      Shoot Diffusion or Wave
```

Comment:
You did add the event right, but what i didnt realize is that you also need to account for the fact the room is heated while you're doing this.

Reply:
hmm, no idea how long the cutscene is or what that damage scales to, i'll whip something up

Reply:
You dont need to care for the cutscene. You wont take damage during it. Because constant damage scaling will be what logic considers in the future, i'd say just put in Varia for now.